llama-index-integrations/vector_stores/llama-index-vector-stores-lancedb/pyproject.toml

```diff
@@ -32,7 +32,7 @@ version = "0.1.5"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.10.1"
-lancedb = "^0.5.1"
+lancedb = ">=0.6.13"
```

Comment:
should we vbump the overall integration package as well?

Reply:
I left it as the previous version as the latest llamaindex-lancedb pypi version was launched on june7 , lancedb is currently 0.8.2 but has dropped support for python3.8 , hence I have added 0.6.13 to pass CI checks, If its okay can we merge it for now?

Reply:
yea sure, that just means these changes won't be published

Reply:
We can always drop support for 3.8 in the integration here is well. Many integrations only support python3.9 or 3.10 as well

Reply:
oh okay then , let me bump up the version Thanks

Reply:
@logan-markewich , I have updated the version to 0.1.6 and bumped python requirement to >=3.9, can we drop support for 3.8? Thanks

Reply:
Yea I just need to push a change to this branch to disable 3.8 tests for this package (need to get back to my laptop for that 😅)